JSDemos/Demos/Charts/AutoCalculatedBarWidth/description.md

```diff
@@ -1 +1,3 @@
-When values for a&nbsp;series are not specified or&nbsp;are set to&nbsp;zero, bars that represent these values will not be&nbsp;visible within the chart. You can populate these empty region by&nbsp;extending neighboring bars via the **equalBarWidth** property set to _false_.
\ No newline at end of file
+When you have multiple [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/), values for some of their points can be unspecified or set to zero. Bars for these points are not displayed, but space for them remains reserved. This results in gaps.
+
+The [ignoreEmptyPoints](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/#ignoreEmptyPoints) option allows you to prevent this behavior. If you set it to **true**, the neighbouring bars increase their width, thus filling the gaps. In this demo, you can see the option's effect on the example of the `Iran`, `Canada`, `Saudi Arabia`, and `Mexico` arguments.
```

Comment:
I wouldn't say "prevent". It's really not a bad thing and sometimes maybe I'd choose the original behavior. So maybe "change" is a better word. 

I'm not sure what you meant by "neighboring". All these bars are neighboring relative to some other bars :-) 

```suggestion
The [ignoreEmptyPoints](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Series_Types/BarSeries/#ignoreEmptyPoints) option allows you to change this behavior. If you set it to **true**, bars increase their width to fill the gaps. In this demo, you can see how the option affects the following arguments: `Iran`, `Canada`, `Saudi Arabia`, and `Mexico`.
```